packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -400,6 +414,28 @@ export class LoadBalancer extends Resource implements IConnectable {
   }
 }
 
+/**
+ * An EC2 instance that is the target for load balancing
+ *
+ */
+export class InstanceTarget implements ILoadBalancerTarget {
+  readonly connections: Connections;
+  readonly ec2Instance?: Instance
+  /**
+   * Create a new Instance target.
+   *
+   * @param instance Instance to register to.
+   * @param port Override the default port for the target.
+   */
+  constructor(public readonly instance: Instance, public readonly port: number) {
+    this.connections = instance.connections;
+    this.ec2Instance = instance;
+  }
+  public attachToClassicLB(_loadBalancer: LoadBalancer): void {
```

Comment:
```suggestion
  public attachToClassicLB(loadBalancer: LoadBalancer): void {
```

The `_` is usually used to describe a property that has to be in the function definition, but not used in the function. Usually the case is when you are implementing an inherited function but some properties in the definition are not useful for your particular implementation. Here, you are using `loadBalancer`, so there's no need to add `_` as a prefix.